tests/test_api_longtests.py

```diff
@@ -54,7 +67,14 @@ def test_list_issue(instance):
 
 
 def test_list_team_members(instance):
-    org = Organization.request(instance, test_org)
+    orgs = instance.get_orgs()
```

Comment:
Does this make sense to move into a helper function?

Reply:
Yeah, I think a better method is to create a unique repo for each test

Reply:
lmk if you like these changes - it would be a lot more work to disentangle the tests in `test_api.py`, but hopefully this is a move in the right direction